src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx

```diff
@@ -5,12 +5,19 @@ import { useEffect, useState } from "react";
 import {
   EvmBridgeConfig,
   getСhainTransferFallbackConfig,
+  chainbridgeConfig,
 } from "../../../chainbridgeConfig";
 import { useNetworkManager } from "../../NetworkManagerContext";
 import { IDestinationBridgeProviderProps } from "../interfaces";
 import { DestinationBridgeContext } from "../../DestinationBridgeContext";
 import { getProvider, getErc20ProposalHash, VoteStatus } from "./helpers";
 import { Fallback } from "../../../Utils/Fallback";
+import { GA } from "../../../Utils/GA";
+const ga = new GA({
```

Comment:
Let's create an `AnalyticsService` and encapsulate all details (event name, parameters structure, etc.) there so in business code you need to trigger a method with required parameters.

Reply:
This service was designed as a reusable utils library that can be used inside any application. So its encapsulation borders are correct in that terms. I didn't have a purpose to create some wrap around it and designed it to be universal and simple.

Reply:
fixed